tests/unit_test/action/kremote_action_test.py

```
@@ -0,0 +1,77 @@
+import pytest
+from aioresponses import aioresponses
+from kairon.chat.actions import KRemoteAction
+from rasa.utils.endpoints import EndpointConfig
+from rasa.utils.endpoints import ClientResponseError
+import ssl
```

Ruff

`ssl` imported but unused

Remove unused import: `ssl`

(F401)